lib/sda/host.go

```diff
@@ -175,7 +175,7 @@ func (h *Host) Listen() {
 		dbg.Lvl3(h.workingAddress, "Accepted Connection from", c.Remote())
 		// register the connection once we know it's ok
 		h.registerConnection(c)
-		h.handleConn(c)
+		go h.handleConn(c)
```

Comment:
You can remove the go as the function is already supposed to happens in a separate goroutine (at least in my lib/network version)